script/Ciel/Day 14/14_01B_03_CIEL14B_4_1C.txt

```diff
@@ -393,30 +434,32 @@ This time, I win.
 [sha:256b83f0955f98f7b776f24d67e6f89f15b77f8a]{
 -- Page 66, Offset 40443.
 -- 「“――――――ああ”」
-"――――――Aah."
+"Ah―――"
 }
 [sha:be199b5d2412680e12820ac43adcc51fa8df3686]{
 -- Page 66, Offset 40444.
 -- 眼球が燃える。振り上げた右手が軋む。
-My eyeballs burn. My overhead right hand creaks.
+My eyes are on fire.
+ My raised right hand groans from the stress.
 }
 [sha:c77092b01144a355991fb121af79067d5266dc87]{
 -- Page 66, Offset 40445.
 -- 地脈の命とも言える“点”。
-The 'point' that marks the life of the earth line.
+This 'point' represents the life running through the Earth's veins.
 }
 [sha:f1791bc949da02d7743c8362cf63d3ca0c34d9bd]{
 -- Page 66, Offset 40446.
 -- この一帯の世界そのものの『死』に、俺はナイフを、
-I plunge the knife into the 'death' of this very world...
+I angle my knife toward the 'death' of the world around me―――
 }
 [sha:bdc7c9c2f1a5cadf5eb3db9fe0c27df0126517dc]{
 -- Page 66, Offset 40447.
 -- 「“―――そういうこと？”」
-"―――Is that what this is about?"
+"―――So that's what you were up to."
 }
 [sha:79cc910686ffb6258e7372b6132190e27f98440a]{
 -- Page 67, Offset 40448.
 -- ―――ナイフは、宙を、切った。
-―――The knife cuts through space.
+// Feels like a spiritual continuation of 40446, but new page, so kept as a standalone sentence
+―――My knife comes slicing down.
```

Comment:
I get the feeling it's supposed to be like "I cut at nothing" since he misses, but it sounds vaguely like he's just readying his swing, so the reader gets caught by the tweest that actually he didn't kill shit.
Guess to preserve both of those impression, something like this?
>My knife cuts through the nothingness.